packages/mui-material/src/FormLabel/FormLabel.d.ts

```diff
@@ -71,6 +72,11 @@ declare const FormLabel: OverridableComponent<FormLabelTypeMap>;
 
 export type FormLabelBaseProps = React.LabelHTMLAttributes<HTMLLabelElement>;
 
+export interface ExtendFormLabelTypeMap<M extends OverridableTypeMap> {
+  props: M['props'] & FormLabelOwnProps;
```

Comment:
Along with also added `color` prop because `color` prop description for `InputLabel` is derived from `FormLabel` `color` prop.

https://github.com/mui/material-ui/pull/35852/files#diff-40c72c413c58d6ccab3754f4c54a76b05d428fffc892c85f66bae5b290934912R76